engine/apps/schedules/tasks/notify_about_empty_shifts_in_schedule.py

```diff
@@ -91,8 +91,8 @@ def notify_about_empty_shifts_in_schedule(schedule_pk):
             f"The user should have Editor or Admin access.\n\n"
         )
         for idx, empty_shift in enumerate(empty_shifts):
-            start_timestamp = int(empty_shift.start.astimezone(pytz.UTC).timestamp())
-            end_timestamp = int(empty_shift.end.astimezone(pytz.UTC).timestamp())
+            start_timestamp = empty_shift.start.astimezone(pytz.UTC).timestamp()
+            end_timestamp = empty_shift.end.astimezone(pytz.UTC).timestamp()
```

Comment:
`format_datetime_to_slack` is typed to take a `float` as its first argument, however this isn't true, you need to pass an `int` or else you get an error.

Since `datetime.datetime.timestamp()` returns a `float`, lets just do the `int` conversion within `format_datetime_to_slack`.